contracts/Core/StakeManager.sol

```diff
@@ -312,6 +312,19 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         emit Withdrew(epoch, stakerId, rAmount, staker.stake, block.timestamp);
     }
 
+    /// @notice remove all funds in case of emergency
```

Comment:
Probably worth it to add more info as comments here. 

"Remove all funds in case of emergency, to safeguard users funds in case of a bug"